blocks/masthead-block/features/masthead-block/masthead-block.scss

```
@@ -1,4 +1,10 @@
 .masthead-block--text {
   font-size: calculateRem(14px);
   line-height: calculateRem(16px);
+
```

this isn't working as expected?
```suggestion
   color: #191919;
   text-decoration: none;
```